tests/unit/conftest.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""pytest fixtures for the unit test."""
+
+import contextlib
+import typing
+import unittest.mock
+
+import ops
+import pytest
+from ops.model import BlockedStatus, Container
+from ops.testing import Harness
+
+from tests.unit import helpers
+from tests.unit._patched_charm import DiscourseCharm, pgsql_patch
+
+BLOCKED_STATUS = BlockedStatus.name  # type: ignore
```

Comment:
@benhoyt this shouldn't be needed anymore, right? Or is that change not released yet?

Reply:
I'm not quite sure which change you're referring to. However, from a quick grep, it doesn't look like this value is actually used. It's defined separately in `tests/unit/test_charm.py` and used once there as follows:

```python
    assert harness.model.unit.status.name == BLOCKED_STATUS
```

However, I think the better and more Pythonic way to write that is to use `isinstance`:

```python
    assert isinstance(harness.model.unit.status, ops.BlockedStatus)
```

Reply:
Thanks Ben

Besides not being used, the reason this `type: ignore` used to be needed is because `BlockedStatus.name` and the `.name` attribute of other statuses used to be `None | str`, I believe it was `None` in the base class. I remember a PR that changed that

Reply:
I rewrote this status check the way Ben proposed.